user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Order_DataMiner_credits
+---
+
+# Order DataMiner credits
+
+1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and Search for DataMiner.
+    ![Azure Marketplace DataMiner credits offer](~/user-guide/images/Azure_Marketplace_Offer.png)
+
+1. Click on *Get It Now* and continue to Azure portal.
+
+> [!WARNING]
+> make sure you have selected the plan Dataminer credits.
+
+1. Once redirected to the Azure portal click on Subscribe. This will start the process of creating a SaaS subscription resource to be deployed in a resource group.
+
+1. Using your new SaaS subscription click on *Configure account now*. This will redirect you to our marketplace landing page.
+    ![Configure account now](~/user-guide/images/Azure_Portal_Configure.png)
+
+1. After confirming all you details click on *Purchase credits*. This will activate your subscription and may take a couple minutes.
+
+1. Once your subscription is activated, Fill in your Organization ID
+    1. To find your organization ID, Go to [dataminer.services](https://dataminer.services).
+    1. Log in using your dataminer.services account.
+    1. Select the organization you want to transfer DataMiner credits to.
+    1. Go to the webpage URL and copy the last part of the URL after the final slash.
+   ![How to retrieve organization ID](~/user-guide/images/Retrieve_Organization_ID.gif)
+
+1. Use the slider to indicate how many credits you wish to purchase for the organization.
+
+1. Click on *Purchase credits* and confirm your purchase
+
+> [!NOTE]
+> If you would like to order more credits you do not have to create a new SaaS subscription. You can simply go to your SaaS subscription in the Azure portal and click on *Open SaaS Account on publisher's site* to be redirected again to our page.
```

Comment:
```suggestion
> If you would like to order more credits, you do not have to create a new SaaS subscription. You can simply go to your SaaS subscription in the Azure portal and click on *Open SaaS Account on publisher's site* to be redirected to our page again.
```

Reply:
Maybe find a better way to say "our page"? Like purchase/order page or something?